### ISSUE TITLE
Curve not followes until I modify it

### ISSUE THREAD

**Issue by disco79stu** [Rem0o/FanControl.Releases#2522]
I noticed that most of the time when I boot up the PC, the fans are spinning higher than they should (with just windows running). When I then go into my fan control curve and modify something (doesn't matter what, can be a tiny change in the highest curve point etc.), only then the fans seem to follow the curve settings and decelerate significantly. It's like sometimes they won't use the curve unless I open-modify-close it.

**Comment by Marius8** [comment]
sounds like another program is overwriting the FanControl values, until you save it..
do you have any other program from your board vendor (e.g. smart fan for Gigabyte) that starts after FanControl?

**Comment by disco79stu** [comment]
> sounds like another program is overwriting the FanControl values, until you save it.. do you have any other program from your board vendor (e.g. smart fan for Gigabyte) that starts after FanControl?

Yes, iCue is running as well and controls the CPU fans (only), because I couldn't FanControl to control the CPU fans. I admit, it's a bit of a mess but that's the only way I could make it work. Some casing fans seem to be connected, others are not recognized at all etc... And FanControl can't recognize the speeds (so it says) but all in all it works to keep noise down and fans running depending on the CPU and system temps.

But I guess that's the reason for the above described issues then...

**Comment by Rem0o** [comment]
Have you taken a look at: https://github.com/EvanMulawski/FanControl.CorsairLink ?

This brings ICUE hardware support to fancontrol, meaning you can ditch ICue all together.

**Comment by disco79stu** [comment]
> Have you taken a look at: https://github.com/EvanMulawski/FanControl.CorsairLink ?
> 
> This brings ICUE hardware support to fancontrol, meaning you can ditch ICue all together.

Thanks, This actually cleaned my situation up a bit. But there are still some fans not connecting properly (or redundant) and teh CPU fans (there are two) don't seem to be recognized either...

![image](https://github.com/Rem0o/FanControl.Releases/assets/164195731/19b50f91-8a3f-46d1-b067-3cc7f1f26a67)

**Comment by Rem0o** [comment]
You need to figure out where those fans are plugged in. They should be on your motherboard headers.

I guess you ran the assisted setup over again ?

**Comment by disco79stu** [comment]
> You need to figure out where those fans are plugged in. They should be on your motherboard headers.
> 
> I guess you ran the assisted setup over again ?

Yes, I ran the assisted setup. 
I will have to check how the fans are connected then, not sure I will understand this, but I will do my best. 
I'm also getting this:
![image](https://github.com/Rem0o/FanControl.Releases/assets/164195731/d0a2a7c0-6465-4045-9005-bc648d7e55bc)

**Comment by Rem0o** [comment]
Read this carefully. You might have missed something.

https://github.com/EvanMulawski/FanControl.CorsairLink#installation

**Comment by disco79stu** [comment]
Hm, I did, Can't see what I might have missed. Removed iCue, no Corsair Service is running, Unblocked the file, copied it to Plugins folder, installed the driver.
I refreshed the whole thing again and I get a bunch of sensor errors. Corsair Links it seems isn't working because of this.
![image](https://github.com/Rem0o/FanControl.Releases/assets/164195731/e0916a62-7229-4256-8d37-c6ed54371de9)

**Comment by disco79stu** [comment]
Still not working, and the initialize-error re-appeared. 
It won't find a sensor for the CPU fans and finds one fan that seems to be showing a phantom speed because no fan or the measurements reacts when I change the speed. I'm starting to thing my system build has some flaws.
